src/main/webapp/app/iris/state-store.service.ts

```diff
@@ -203,14 +195,14 @@ export class IrisStateStore implements OnDestroy {
                 return {
                     ...state,
                     isLoading: true,
-                    error: null,
+                    error: castedAction.message.id === undefined ? defaultError : null,
```

Comment:
Why not just set it to `defaultError`?

Reply:
I did this to preserve the current behavior, where the error stays, as we received an already known message over the websocket.